### ISSUE TITLE
[Bug] Pybricks Beta does not overwrite existing file on import if requested to overwrite

### ISSUE THREAD

**Issue by BertLindeman** [pybricks/support#975]
**Describe the bug**
Import of file already existing in beta.pybricks.com neatly shows dialog.
But a press on the red replace button does not cause the imported file to be really loaded.

**To reproduce**

In beta.pybricks.com add a new program:
``` python
print("original program")
```

Save the program in `Z:\home\bert\py\pybricks\issue`
as `try_overwrite.py`

Update the program in beta.pybricks.com like this:
``` python
print("original program")
print("after save: updated")
```

Now import the original saved program.

dialog shows: 
![afbeelding](https://user-images.githubusercontent.com/8142081/223988676-91a59455-6f74-40db-92b2-56a8a4590d94.png)

Press the red replace button.
No error shown.

Close  `try_overwrite.py`  in the active tab and
re-open `try_overwrite.py` and see still the updated source, **not** the original saved one (as was saved).

Screenshot:
![afbeelding](https://user-images.githubusercontent.com/8142081/223994146-775d6ed0-b2db-4c85-8f1b-08150c4e140f.png)

**Expected behavior**
See newly imported version of source.

**Environment**
Pybricks v3.3.0b2 (Pybricks Beta v2.2.0-beta.1)
Windows 11 Ver 22H2
Microsoft Edge Version 110.0.1587.63 (Official build) (64-bit)

EDIT: typo (still wrong in description "exsisting")

**Comment by BertLindeman** [comment]
Huh??

If I set and unset the "remember" flag on the dialog and **then** press the red replace button, **then** the replace works.

If I do not touch the "remember" flag on the dialog, the replace is not done, each time.
No message seen in developers console.

**Comment by BertLindeman** [comment]
If the program is open as programming tab then the replace also does not work
regardless op the remember flag.
(hope this is a bit understandable jargon)

**Comment by dlech** [comment]
I think I got this fixed in Pybricks Code v2.2.0-beta.5. At least the case replacing a file that is open in the editor. Let us know if you still have other issues with this.

**Comment by BertLindeman** [comment]
Works for me.
If asking to overwrite, file is overwritten like requested, regardless the remember-flag.
First tested if open in the editor.
Also if the file is NOT open in the editor, all goes as expected, too.

I am just not sure that the remember flag is indeed remembered.
But this issue has been solved at Pybricks Code v2.2.0-beta.5.

And not related but good to see copied from the status bar:
<html>
<body>


Connected to: | spikeBeta
-- | --
Hub type: | Prime hub
Firmware: | v3.3.0b5


</body>
</html>
